File: BaekJoon Online Judge/C++/Step09-Math_2 Prime and Geometric/4153.cpp
//
//  4153.cpp
//  Step9-Math_2 Prime and Geometric
//
//  Created by 김예빈 on 2019. 9. 5..
//  Copyright © 2019년 김예빈. All rights reserved.
//

#include <iostream>
using namespace std;

int main(int argc, const char * argv[]) {
    cin.tie(NULL);
    ios::sync_with_stdio(false);
    
    int a, b, c;
    
    while(1) {
        cin >> a >> b >> c;
        
        if(!a && !b && !c)
            break;
        
        a *= a;
        b *= b;
        c *= c;
        
        if(a + b == c || a + c == b || b + c == a)
            cout << "right\n";
        else
            cout << "wrong\n";
    }
    return 0;
}

File: BaekJoon Online Judge/Swift/Step09-Math_2 Prime and Geometric/4153.swift
//
//  4153.swift
//  Step09-Math_2 Prime and Geometric
//
//  Created by Yebin Kim on 2019/11/22.
//  Copyright © 2019 김예빈. All rights reserved.
//

import Foundation

while true {
    let triangle = readLine()!.split(separator: " ").map{ Int($0)! * Int($0)! }.sorted()
    
    let bp = triangle.reduce(0){ $0 + $1 }
    if bp == 0 { break }
    
    if triangle[2] == triangle[1] + triangle[0] {
        print("right")
    } else {
        print("wrong")
    }
}
